File: Document Intelligence/Analyze.py
endpoint = "YOUR_DOC_INTELLIGENCE_ENDPOINT"

key = "YOUR_DOC_INTELLIGENCE_KEY"

 

model_id = "YOUR_CUSTOM_BUILT_MODEL_ID"

formUrl = "YOUR_DOCUMENT"

 

document_analysis_client = DocumentAnalysisClient(

    endpoint=endpoint, credential=AzureKeyCredential(key)

)

 

# Make sure your document's type is included in the list of document types the custom model can analyze

task = document_analysis_client.begin_analyze_document_from_url(model_id, formUrl)

result = task.result()

File: Document Intelligence/Analyze.cs
string endpoint = "<endpoint>";

string apiKey = "<apiKey>";

AzureKeyCredential credential = new AzureKeyCredential(apiKey);

DocumentAnalysisClient client = new DocumentAnalysisClient(new Uri(endpoint), credential);

 

string modelId = "<modelId>";

Uri fileUri = new Uri("<fileUri>");

 

AnalyzeDocumentOperation operation = await client.AnalyzeDocumentFromUriAsync(WaitUntil.Completed, modelId, fileUri);

AnalyzeResult result = operation.Value;
